hrjobcontract/js/src/job-contract/controllers/form/form-pay.js

```diff
@@ -1 +1,167 @@
                entityPay.pay_currency = '';
                entityPay.pay_annualized_est = '';
                entityPay.pay_is_auto_est = '';
                entityPay.annual_benefits = [];
                entityPay.annual_deductions = [];
                entityPay.pay_cycle = '';
                entityPay.pay_per_cycle_gross = '';
                entityPay.pay_per_cycle_net = '';
                $scope.benefits_per_cycle = '';
                $scope.deductions_per_cycle = '';
            };

            $scope.setDefaults = function(){
                entityPay.pay_cycle = defaults.pay_cycle;
                entityPay.pay_currency = defaults.pay_currency;
                entityPay.pay_is_auto_est = '0';
                entityPay.pay_amount = (0).toFixed(2);
            };

            $scope.remove = function(array, index){
                array.splice(index, 1);
            };

            $scope.$watch('entity.pay.pay_amount', $scope.calcAnnualPayEst);
            $scope.$watch('entity.pay.pay_unit', $scope.calcAnnualPayEst);
            $scope.$watch('entity.pay.pay_annualized_est', function(){
                $scope.calcPayPerCycleGross();
                $scope.calcBenefitsPerCycle();
                $scope.calcDeductionsPerCycle();
            });
            $scope.$watch('entity.pay.annual_benefits', $scope.calcBenefitsPerCycle,true);
            $scope.$watch('entity.pay.annual_deductions', $scope.calcDeductionsPerCycle,true);
            $scope.$watch('benefits_per_cycle',$scope.calcBenefitsPerCycleNet);
            $scope.$watch('deductions_per_cycle',$scope.calcBenefitsPerCycleNet);
            $scope.$watch('benefits_per_cycle_net',$scope.calcPayPerCycleNet);
            $scope.$watch('entity.pay.pay_per_cycle_gross',$scope.calcPayPerCycleNet);

        }]);
});
\ No newline at end of file
+define([
+    'job-contract/controllers/controllers'
+], function (controllers) {
+    'use strict';
+
+    controllers.controller('FormPayCtrl',['$scope','$filter','settings', '$log',
+        function ($scope, $filter, settings, $log) {
+            $log.debug('Controller: FormPayCtrl');
+
+            var entityPay = $scope.entity.pay || {},
+                defaults = {
+                    pay_amount: 0,
+                    pay_currency: settings.CRM.defaultCurrency,
+                    pay_cycle: '2',
+                    pay_unit: 'Year'
+                },
+                utilsPayScaleGrade = $scope.utils.payScaleGrade,
+                workPerYear = {
+                    Year: 1,
+                    Month: 12,
+                    Week: 52,
+                    Fortnight: 26,
+                    Day: 260,
+                    Hour: 2080
+                };
+
+            entityPay.is_paid = entityPay.is_paid || 0;
+            entityPay.pay_is_auto_est = '0';
+            entityPay.annual_benefits = entityPay.annual_benefits || [];
+            entityPay.annual_deductions = entityPay.annual_deductions || [];
```

Comment:
Unfortunately the whole file is diff'd since the line ending had been changed from MacOS to UNIX. Anyway the change consists in lines 27-30 having been moved from below line 35